diff --git a/cspice/src/brief_c/mkprodct.csh b/cspice/src/brief_c/mkprodct.csh
index d7232cd..bc35a67 100755
--- a/cspice/src/brief_c/mkprodct.csh
+++ b/cspice/src/brief_c/mkprodct.csh
@@ -160,7 +160,7 @@ else
 #                        implementation
 #
 #
-   set TKCOMPILEOPTIONS = "-c -ansi -m64 -O2 -DNON_UNIX_STDIO"
+   set TKCOMPILEOPTIONS = "-c -ansi -O2 -DNON_UNIX_STDIO"
    echo " "
    echo "      Setting default compile options:"
    echo "      $TKCOMPILEOPTIONS"
@@ -171,7 +171,7 @@ if ( $?TKLINKOPTIONS ) then
    echo "      Using link options: "
    echo "      $TKLINKOPTIONS"
 else
-   set TKLINKOPTIONS = "-lm -m64"
+   set TKLINKOPTIONS = "-lm"
    echo " "
    echo "      Setting default link options:"
    echo "      $TKLINKOPTIONS"
diff --git a/cspice/src/chrnos_c/mkprodct.csh b/cspice/src/chrnos_c/mkprodct.csh
index d7232cd..bc35a67 100755
--- a/cspice/src/chrnos_c/mkprodct.csh
+++ b/cspice/src/chrnos_c/mkprodct.csh
@@ -160,7 +160,7 @@ else
 #                        implementation
 #
 #
-   set TKCOMPILEOPTIONS = "-c -ansi -m64 -O2 -DNON_UNIX_STDIO"
+   set TKCOMPILEOPTIONS = "-c -ansi -O2 -DNON_UNIX_STDIO"
    echo " "
    echo "      Setting default compile options:"
    echo "      $TKCOMPILEOPTIONS"
@@ -171,7 +171,7 @@ if ( $?TKLINKOPTIONS ) then
    echo "      Using link options: "
    echo "      $TKLINKOPTIONS"
 else
-   set TKLINKOPTIONS = "-lm -m64"
+   set TKLINKOPTIONS = "-lm"
    echo " "
    echo "      Setting default link options:"
    echo "      $TKLINKOPTIONS"
diff --git a/cspice/src/ckbref_c/mkprodct.csh b/cspice/src/ckbref_c/mkprodct.csh
index d7232cd..bc35a67 100755
--- a/cspice/src/ckbref_c/mkprodct.csh
+++ b/cspice/src/ckbref_c/mkprodct.csh
@@ -160,7 +160,7 @@ else
 #                        implementation
 #
 #
-   set TKCOMPILEOPTIONS = "-c -ansi -m64 -O2 -DNON_UNIX_STDIO"
+   set TKCOMPILEOPTIONS = "-c -ansi -O2 -DNON_UNIX_STDIO"
    echo " "
    echo "      Setting default compile options:"
    echo "      $TKCOMPILEOPTIONS"
@@ -171,7 +171,7 @@ if ( $?TKLINKOPTIONS ) then
    echo "      Using link options: "
    echo "      $TKLINKOPTIONS"
 else
-   set TKLINKOPTIONS = "-lm -m64"
+   set TKLINKOPTIONS = "-lm"
    echo " "
    echo "      Setting default link options:"
    echo "      $TKLINKOPTIONS"
diff --git a/cspice/src/commnt_c/mkprodct.csh b/cspice/src/commnt_c/mkprodct.csh
index d7232cd..bc35a67 100755
--- a/cspice/src/commnt_c/mkprodct.csh
+++ b/cspice/src/commnt_c/mkprodct.csh
@@ -160,7 +160,7 @@ else
 #                        implementation
 #
 #
-   set TKCOMPILEOPTIONS = "-c -ansi -m64 -O2 -DNON_UNIX_STDIO"
+   set TKCOMPILEOPTIONS = "-c -ansi -O2 -DNON_UNIX_STDIO"
    echo " "
    echo "      Setting default compile options:"
    echo "      $TKCOMPILEOPTIONS"
@@ -171,7 +171,7 @@ if ( $?TKLINKOPTIONS ) then
    echo "      Using link options: "
    echo "      $TKLINKOPTIONS"
 else
-   set TKLINKOPTIONS = "-lm -m64"
+   set TKLINKOPTIONS = "-lm"
    echo " "
    echo "      Setting default link options:"
    echo "      $TKLINKOPTIONS"
diff --git a/cspice/src/cook_c/mkprodct.csh b/cspice/src/cook_c/mkprodct.csh
index 2a49ee7..9653500 100755
--- a/cspice/src/cook_c/mkprodct.csh
+++ b/cspice/src/cook_c/mkprodct.csh
@@ -104,7 +104,7 @@ else
 #                        implementation
 #
 #
-   set TKCOMPILEOPTIONS = "-c -ansi -m64 -O2 -DNON_UNIX_STDIO"
+   set TKCOMPILEOPTIONS = "-c -ansi -O2 -DNON_UNIX_STDIO"
    echo " "
    echo "      Setting default compile options:"
    echo "      $TKCOMPILEOPTIONS"
@@ -115,7 +115,7 @@ if ( $?TKLINKOPTIONS ) then
    echo "      Using link options: "
    echo "      $TKLINKOPTIONS"
 else
-   set TKLINKOPTIONS = "-lm -m64"
+   set TKLINKOPTIONS = "-lm"
    echo " "
    echo "      Setting default link options:"
    echo "      $TKLINKOPTIONS"
diff --git a/cspice/src/cspice/mkprodct.csh b/cspice/src/cspice/mkprodct.csh
index e08a643..929c5b6 100755
--- a/cspice/src/cspice/mkprodct.csh
+++ b/cspice/src/cspice/mkprodct.csh
@@ -169,7 +169,7 @@ else
 #                        implementation
 #
 #
-   set TKCOMPILEOPTIONS = "-c -ansi -m64 -O2 -fPIC -DNON_UNIX_STDIO"
+   set TKCOMPILEOPTIONS = "-c -ansi -O2 -fPIC -DNON_UNIX_STDIO"
    echo " "
    echo "      Setting default compile options:"
    echo "      $TKCOMPILEOPTIONS"
@@ -180,7 +180,7 @@ if ( $?TKLINKOPTIONS ) then
    echo "      Using link options: "
    echo "      $TKLINKOPTIONS"
 else
-   set TKLINKOPTIONS = "-lm -m64"
+   set TKLINKOPTIONS = "-lm"
    echo " "
    echo "      Setting default link options:"
    echo "      $TKLINKOPTIONS"
diff --git a/cspice/src/csupport/mkprodct.csh b/cspice/src/csupport/mkprodct.csh
index 2244d9a..5639a39 100755
--- a/cspice/src/csupport/mkprodct.csh
+++ b/cspice/src/csupport/mkprodct.csh
@@ -165,7 +165,7 @@ else
 #
 #     -fPIC              position-independent code
 #
-   set TKCOMPILEOPTIONS = "-c -ansi -m64 -O2 -fPIC -DNON_UNIX_STDIO"
+   set TKCOMPILEOPTIONS = "-c -ansi -O2 -fPIC -DNON_UNIX_STDIO"
    echo " "
    echo "      Setting default compile options:"
    echo "      $TKCOMPILEOPTIONS"
@@ -176,7 +176,7 @@ if ( $?TKLINKOPTIONS ) then
    echo "      Using link options: "
    echo "      $TKLINKOPTIONS"
 else
-   set TKLINKOPTIONS = "-lm -m64"
+   set TKLINKOPTIONS = "-lm"
    echo " "
    echo "      Setting default link options:"
    echo "      $TKLINKOPTIONS"
diff --git a/cspice/src/dskbrief_c/mkprodct.csh b/cspice/src/dskbrief_c/mkprodct.csh
index 74c5584..a2f5a93 100755
--- a/cspice/src/dskbrief_c/mkprodct.csh
+++ b/cspice/src/dskbrief_c/mkprodct.csh
@@ -160,7 +160,7 @@ else
 #                        implementation
 #
 #
-   set TKCOMPILEOPTIONS = "-c -ansi -m64 -O2 -DNON_UNIX_STDIO"
+   set TKCOMPILEOPTIONS = "-c -ansi -O2 -DNON_UNIX_STDIO"
    echo " "
    echo "      Setting default compile options:"
    echo "      $TKCOMPILEOPTIONS"
@@ -171,7 +171,7 @@ if ( $?TKLINKOPTIONS ) then
    echo "      Using link options: "
    echo "      $TKLINKOPTIONS"
 else
-   set TKLINKOPTIONS = "-lm -m64"
+   set TKLINKOPTIONS = "-lm"
    echo " "
    echo "      Setting default link options:"
    echo "      $TKLINKOPTIONS"
diff --git a/cspice/src/dskexp_c/mkprodct.csh b/cspice/src/dskexp_c/mkprodct.csh
index 74c5584..a2f5a93 100755
--- a/cspice/src/dskexp_c/mkprodct.csh
+++ b/cspice/src/dskexp_c/mkprodct.csh
@@ -160,7 +160,7 @@ else
 #                        implementation
 #
 #
-   set TKCOMPILEOPTIONS = "-c -ansi -m64 -O2 -DNON_UNIX_STDIO"
+   set TKCOMPILEOPTIONS = "-c -ansi -O2 -DNON_UNIX_STDIO"
    echo " "
    echo "      Setting default compile options:"
    echo "      $TKCOMPILEOPTIONS"
@@ -171,7 +171,7 @@ if ( $?TKLINKOPTIONS ) then
    echo "      Using link options: "
    echo "      $TKLINKOPTIONS"
 else
-   set TKLINKOPTIONS = "-lm -m64"
+   set TKLINKOPTIONS = "-lm"
    echo " "
    echo "      Setting default link options:"
    echo "      $TKLINKOPTIONS"
diff --git a/cspice/src/frmdif_c/mkprodct.csh b/cspice/src/frmdif_c/mkprodct.csh
index d7232cd..bc35a67 100755
--- a/cspice/src/frmdif_c/mkprodct.csh
+++ b/cspice/src/frmdif_c/mkprodct.csh
@@ -160,7 +160,7 @@ else
 #                        implementation
 #
 #
-   set TKCOMPILEOPTIONS = "-c -ansi -m64 -O2 -DNON_UNIX_STDIO"
+   set TKCOMPILEOPTIONS = "-c -ansi -O2 -DNON_UNIX_STDIO"
    echo " "
    echo "      Setting default compile options:"
    echo "      $TKCOMPILEOPTIONS"
@@ -171,7 +171,7 @@ if ( $?TKLINKOPTIONS ) then
    echo "      Using link options: "
    echo "      $TKLINKOPTIONS"
 else
-   set TKLINKOPTIONS = "-lm -m64"
+   set TKLINKOPTIONS = "-lm"
    echo " "
    echo "      Setting default link options:"
    echo "      $TKLINKOPTIONS"
diff --git a/cspice/src/inspkt_c/mkprodct.csh b/cspice/src/inspkt_c/mkprodct.csh
index d7232cd..bc35a67 100755
--- a/cspice/src/inspkt_c/mkprodct.csh
+++ b/cspice/src/inspkt_c/mkprodct.csh
@@ -160,7 +160,7 @@ else
 #                        implementation
 #
 #
-   set TKCOMPILEOPTIONS = "-c -ansi -m64 -O2 -DNON_UNIX_STDIO"
+   set TKCOMPILEOPTIONS = "-c -ansi -O2 -DNON_UNIX_STDIO"
    echo " "
    echo "      Setting default compile options:"
    echo "      $TKCOMPILEOPTIONS"
@@ -171,7 +171,7 @@ if ( $?TKLINKOPTIONS ) then
    echo "      Using link options: "
    echo "      $TKLINKOPTIONS"
 else
-   set TKLINKOPTIONS = "-lm -m64"
+   set TKLINKOPTIONS = "-lm"
    echo " "
    echo "      Setting default link options:"
    echo "      $TKLINKOPTIONS"
diff --git a/cspice/src/mkdsk_c/mkprodct.csh b/cspice/src/mkdsk_c/mkprodct.csh
index 74c5584..a2f5a93 100755
--- a/cspice/src/mkdsk_c/mkprodct.csh
+++ b/cspice/src/mkdsk_c/mkprodct.csh
@@ -160,7 +160,7 @@ else
 #                        implementation
 #
 #
-   set TKCOMPILEOPTIONS = "-c -ansi -m64 -O2 -DNON_UNIX_STDIO"
+   set TKCOMPILEOPTIONS = "-c -ansi -O2 -DNON_UNIX_STDIO"
    echo " "
    echo "      Setting default compile options:"
    echo "      $TKCOMPILEOPTIONS"
@@ -171,7 +171,7 @@ if ( $?TKLINKOPTIONS ) then
    echo "      Using link options: "
    echo "      $TKLINKOPTIONS"
 else
-   set TKLINKOPTIONS = "-lm -m64"
+   set TKLINKOPTIONS = "-lm"
    echo " "
    echo "      Setting default link options:"
    echo "      $TKLINKOPTIONS"
diff --git a/cspice/src/mkspk_c/mkprodct.csh b/cspice/src/mkspk_c/mkprodct.csh
index d7232cd..bc35a67 100755
--- a/cspice/src/mkspk_c/mkprodct.csh
+++ b/cspice/src/mkspk_c/mkprodct.csh
@@ -160,7 +160,7 @@ else
 #                        implementation
 #
 #
-   set TKCOMPILEOPTIONS = "-c -ansi -m64 -O2 -DNON_UNIX_STDIO"
+   set TKCOMPILEOPTIONS = "-c -ansi -O2 -DNON_UNIX_STDIO"
    echo " "
    echo "      Setting default compile options:"
    echo "      $TKCOMPILEOPTIONS"
@@ -171,7 +171,7 @@ if ( $?TKLINKOPTIONS ) then
    echo "      Using link options: "
    echo "      $TKLINKOPTIONS"
 else
-   set TKLINKOPTIONS = "-lm -m64"
+   set TKLINKOPTIONS = "-lm"
    echo " "
    echo "      Setting default link options:"
    echo "      $TKLINKOPTIONS"
diff --git a/cspice/src/msopck_c/mkprodct.csh b/cspice/src/msopck_c/mkprodct.csh
index d7232cd..bc35a67 100755
--- a/cspice/src/msopck_c/mkprodct.csh
+++ b/cspice/src/msopck_c/mkprodct.csh
@@ -160,7 +160,7 @@ else
 #                        implementation
 #
 #
-   set TKCOMPILEOPTIONS = "-c -ansi -m64 -O2 -DNON_UNIX_STDIO"
+   set TKCOMPILEOPTIONS = "-c -ansi -O2 -DNON_UNIX_STDIO"
    echo " "
    echo "      Setting default compile options:"
    echo "      $TKCOMPILEOPTIONS"
@@ -171,7 +171,7 @@ if ( $?TKLINKOPTIONS ) then
    echo "      Using link options: "
    echo "      $TKLINKOPTIONS"
 else
-   set TKLINKOPTIONS = "-lm -m64"
+   set TKLINKOPTIONS = "-lm"
    echo " "
    echo "      Setting default link options:"
    echo "      $TKLINKOPTIONS"
diff --git a/cspice/src/spacit_c/mkprodct.csh b/cspice/src/spacit_c/mkprodct.csh
index d7232cd..bc35a67 100755
--- a/cspice/src/spacit_c/mkprodct.csh
+++ b/cspice/src/spacit_c/mkprodct.csh
@@ -160,7 +160,7 @@ else
 #                        implementation
 #
 #
-   set TKCOMPILEOPTIONS = "-c -ansi -m64 -O2 -DNON_UNIX_STDIO"
+   set TKCOMPILEOPTIONS = "-c -ansi -O2 -DNON_UNIX_STDIO"
    echo " "
    echo "      Setting default compile options:"
    echo "      $TKCOMPILEOPTIONS"
@@ -171,7 +171,7 @@ if ( $?TKLINKOPTIONS ) then
    echo "      Using link options: "
    echo "      $TKLINKOPTIONS"
 else
-   set TKLINKOPTIONS = "-lm -m64"
+   set TKLINKOPTIONS = "-lm"
    echo " "
    echo "      Setting default link options:"
    echo "      $TKLINKOPTIONS"
diff --git a/cspice/src/spkdif_c/mkprodct.csh b/cspice/src/spkdif_c/mkprodct.csh
index d7232cd..bc35a67 100755
--- a/cspice/src/spkdif_c/mkprodct.csh
+++ b/cspice/src/spkdif_c/mkprodct.csh
@@ -160,7 +160,7 @@ else
 #                        implementation
 #
 #
-   set TKCOMPILEOPTIONS = "-c -ansi -m64 -O2 -DNON_UNIX_STDIO"
+   set TKCOMPILEOPTIONS = "-c -ansi -O2 -DNON_UNIX_STDIO"
    echo " "
    echo "      Setting default compile options:"
    echo "      $TKCOMPILEOPTIONS"
@@ -171,7 +171,7 @@ if ( $?TKLINKOPTIONS ) then
    echo "      Using link options: "
    echo "      $TKLINKOPTIONS"
 else
-   set TKLINKOPTIONS = "-lm -m64"
+   set TKLINKOPTIONS = "-lm"
    echo " "
    echo "      Setting default link options:"
    echo "      $TKLINKOPTIONS"
diff --git a/cspice/src/spkmrg_c/mkprodct.csh b/cspice/src/spkmrg_c/mkprodct.csh
index d7232cd..bc35a67 100755
--- a/cspice/src/spkmrg_c/mkprodct.csh
+++ b/cspice/src/spkmrg_c/mkprodct.csh
@@ -160,7 +160,7 @@ else
 #                        implementation
 #
 #
-   set TKCOMPILEOPTIONS = "-c -ansi -m64 -O2 -DNON_UNIX_STDIO"
+   set TKCOMPILEOPTIONS = "-c -ansi -O2 -DNON_UNIX_STDIO"
    echo " "
    echo "      Setting default compile options:"
    echo "      $TKCOMPILEOPTIONS"
@@ -171,7 +171,7 @@ if ( $?TKLINKOPTIONS ) then
    echo "      Using link options: "
    echo "      $TKLINKOPTIONS"
 else
-   set TKLINKOPTIONS = "-lm -m64"
+   set TKLINKOPTIONS = "-lm"
    echo " "
    echo "      Setting default link options:"
    echo "      $TKLINKOPTIONS"
diff --git a/cspice/src/tobin_c/mkprodct.csh b/cspice/src/tobin_c/mkprodct.csh
index d7232cd..bc35a67 100755
--- a/cspice/src/tobin_c/mkprodct.csh
+++ b/cspice/src/tobin_c/mkprodct.csh
@@ -160,7 +160,7 @@ else
 #                        implementation
 #
 #
-   set TKCOMPILEOPTIONS = "-c -ansi -m64 -O2 -DNON_UNIX_STDIO"
+   set TKCOMPILEOPTIONS = "-c -ansi -O2 -DNON_UNIX_STDIO"
    echo " "
    echo "      Setting default compile options:"
    echo "      $TKCOMPILEOPTIONS"
@@ -171,7 +171,7 @@ if ( $?TKLINKOPTIONS ) then
    echo "      Using link options: "
    echo "      $TKLINKOPTIONS"
 else
-   set TKLINKOPTIONS = "-lm -m64"
+   set TKLINKOPTIONS = "-lm"
    echo " "
    echo "      Setting default link options:"
    echo "      $TKLINKOPTIONS"
diff --git a/cspice/src/toxfr_c/mkprodct.csh b/cspice/src/toxfr_c/mkprodct.csh
index d7232cd..bc35a67 100755
--- a/cspice/src/toxfr_c/mkprodct.csh
+++ b/cspice/src/toxfr_c/mkprodct.csh
@@ -160,7 +160,7 @@ else
 #                        implementation
 #
 #
-   set TKCOMPILEOPTIONS = "-c -ansi -m64 -O2 -DNON_UNIX_STDIO"
+   set TKCOMPILEOPTIONS = "-c -ansi -O2 -DNON_UNIX_STDIO"
    echo " "
    echo "      Setting default compile options:"
    echo "      $TKCOMPILEOPTIONS"
@@ -171,7 +171,7 @@ if ( $?TKLINKOPTIONS ) then
    echo "      Using link options: "
    echo "      $TKLINKOPTIONS"
 else
-   set TKLINKOPTIONS = "-lm -m64"
+   set TKLINKOPTIONS = "-lm"
    echo " "
    echo "      Setting default link options:"
    echo "      $TKLINKOPTIONS"
diff --git a/cspice/src/versn_c/mkprodct.csh b/cspice/src/versn_c/mkprodct.csh
index d7232cd..bc35a67 100755
--- a/cspice/src/versn_c/mkprodct.csh
+++ b/cspice/src/versn_c/mkprodct.csh
@@ -160,7 +160,7 @@ else
 #                        implementation
 #
 #
-   set TKCOMPILEOPTIONS = "-c -ansi -m64 -O2 -DNON_UNIX_STDIO"
+   set TKCOMPILEOPTIONS = "-c -ansi -O2 -DNON_UNIX_STDIO"
    echo " "
    echo "      Setting default compile options:"
    echo "      $TKCOMPILEOPTIONS"
@@ -171,7 +171,7 @@ if ( $?TKLINKOPTIONS ) then
    echo "      Using link options: "
    echo "      $TKLINKOPTIONS"
 else
-   set TKLINKOPTIONS = "-lm -m64"
+   set TKLINKOPTIONS = "-lm"
    echo " "
    echo "      Setting default link options:"
    echo "      $TKLINKOPTIONS"
