commit 4c428ca27890a7fc8e110a4cf57ea22a32aae1dc
Author: Andrew Annex <ama6fy@virginia.edu>
Date:   Sat Apr 10 18:58:49 2021 -0400

    removed m64 flags

diff --git a/cspice/src/brief_c/mkprodct.csh b/cspice/src/brief_c/mkprodct.csh
index 3e49d98..88a43ac 100755
--- a/cspice/src/brief_c/mkprodct.csh
+++ b/cspice/src/brief_c/mkprodct.csh
@@ -145,7 +145,7 @@ else
 #                        implementation
 #
 #
-    set TKCOMPILEOPTIONS = "-m64 -c -ansi -O2 -DNON_UNIX_STDIO"
+    set TKCOMPILEOPTIONS = "-c -ansi -O2 -DNON_UNIX_STDIO"
     echo " "
     echo "      Setting default compile options:"
     echo "      $TKCOMPILEOPTIONS"
@@ -156,7 +156,7 @@ if ( $?TKLINKOPTIONS ) then
     echo "      Using link options: "
     echo "      $TKLINKOPTIONS"
 else
-    set TKLINKOPTIONS = "-m64 -lm"
+    set TKLINKOPTIONS = "-lm"
     echo " "
     echo "      Setting default link options:"
     echo "      $TKLINKOPTIONS"
diff --git a/cspice/src/chrnos_c/mkprodct.csh b/cspice/src/chrnos_c/mkprodct.csh
index 3e49d98..88a43ac 100755
--- a/cspice/src/chrnos_c/mkprodct.csh
+++ b/cspice/src/chrnos_c/mkprodct.csh
@@ -145,7 +145,7 @@ else
 #                        implementation
 #
 #
-    set TKCOMPILEOPTIONS = "-m64 -c -ansi -O2 -DNON_UNIX_STDIO"
+    set TKCOMPILEOPTIONS = "-c -ansi -O2 -DNON_UNIX_STDIO"
     echo " "
     echo "      Setting default compile options:"
     echo "      $TKCOMPILEOPTIONS"
@@ -156,7 +156,7 @@ if ( $?TKLINKOPTIONS ) then
     echo "      Using link options: "
     echo "      $TKLINKOPTIONS"
 else
-    set TKLINKOPTIONS = "-m64 -lm"
+    set TKLINKOPTIONS = "-lm"
     echo " "
     echo "      Setting default link options:"
     echo "      $TKLINKOPTIONS"
diff --git a/cspice/src/ckbref_c/mkprodct.csh b/cspice/src/ckbref_c/mkprodct.csh
index 3e49d98..88a43ac 100755
--- a/cspice/src/ckbref_c/mkprodct.csh
+++ b/cspice/src/ckbref_c/mkprodct.csh
@@ -145,7 +145,7 @@ else
 #                        implementation
 #
 #
-    set TKCOMPILEOPTIONS = "-m64 -c -ansi -O2 -DNON_UNIX_STDIO"
+    set TKCOMPILEOPTIONS = "-c -ansi -O2 -DNON_UNIX_STDIO"
     echo " "
     echo "      Setting default compile options:"
     echo "      $TKCOMPILEOPTIONS"
@@ -156,7 +156,7 @@ if ( $?TKLINKOPTIONS ) then
     echo "      Using link options: "
     echo "      $TKLINKOPTIONS"
 else
-    set TKLINKOPTIONS = "-m64 -lm"
+    set TKLINKOPTIONS = "-lm"
     echo " "
     echo "      Setting default link options:"
     echo "      $TKLINKOPTIONS"
diff --git a/cspice/src/commnt_c/mkprodct.csh b/cspice/src/commnt_c/mkprodct.csh
index 3e49d98..88a43ac 100755
--- a/cspice/src/commnt_c/mkprodct.csh
+++ b/cspice/src/commnt_c/mkprodct.csh
@@ -145,7 +145,7 @@ else
 #                        implementation
 #
 #
-    set TKCOMPILEOPTIONS = "-m64 -c -ansi -O2 -DNON_UNIX_STDIO"
+    set TKCOMPILEOPTIONS = "-c -ansi -O2 -DNON_UNIX_STDIO"
     echo " "
     echo "      Setting default compile options:"
     echo "      $TKCOMPILEOPTIONS"
@@ -156,7 +156,7 @@ if ( $?TKLINKOPTIONS ) then
     echo "      Using link options: "
     echo "      $TKLINKOPTIONS"
 else
-    set TKLINKOPTIONS = "-m64 -lm"
+    set TKLINKOPTIONS = "-lm"
     echo " "
     echo "      Setting default link options:"
     echo "      $TKLINKOPTIONS"
diff --git a/cspice/src/cook_c/mkprodct.csh b/cspice/src/cook_c/mkprodct.csh
index 881d393..fd98fa1 100755
--- a/cspice/src/cook_c/mkprodct.csh
+++ b/cspice/src/cook_c/mkprodct.csh
@@ -110,7 +110,7 @@ else
 #                        implementation
 #
 #
-    set TKCOMPILEOPTIONS = "-m64 -c -ansi -O2 -DNON_UNIX_STDIO"
+    set TKCOMPILEOPTIONS = "-c -ansi -O2 -DNON_UNIX_STDIO"
     echo " "
     echo "      Setting default compile options:"
     echo "      $TKCOMPILEOPTIONS"
@@ -121,7 +121,7 @@ if ( $?TKLINKOPTIONS ) then
     echo "      Using link options: "
     echo "      $TKLINKOPTIONS"
 else
-    set TKLINKOPTIONS = "-m64 -lm"
+    set TKLINKOPTIONS = "-lm"
     echo " "
     echo "      Setting default link options:"
     echo "      $TKLINKOPTIONS"
diff --git a/cspice/src/cspice/mkprodct.csh b/cspice/src/cspice/mkprodct.csh
index 0a75ccb..b49aad7 100755
--- a/cspice/src/cspice/mkprodct.csh
+++ b/cspice/src/cspice/mkprodct.csh
@@ -149,7 +149,7 @@ else
 #                        implementation
 #
 #
-    set TKCOMPILEOPTIONS = "-m64 -c -ansi -O2 -fPIC -DNON_UNIX_STDIO"
+    set TKCOMPILEOPTIONS = "-c -ansi -O2 -fPIC -DNON_UNIX_STDIO"
     echo " "
     echo "      Setting default compile options:"
     echo "      $TKCOMPILEOPTIONS"
@@ -160,7 +160,7 @@ if ( $?TKLINKOPTIONS ) then
     echo "      Using link options: "
     echo "      $TKLINKOPTIONS"
 else
-    set TKLINKOPTIONS = "-m64 -lm"
+    set TKLINKOPTIONS = "-lm"
     echo " "
     echo "      Setting default link options:"
     echo "      $TKLINKOPTIONS"
diff --git a/cspice/src/csupport/mkprodct.csh b/cspice/src/csupport/mkprodct.csh
index b640186..d069926 100755
--- a/cspice/src/csupport/mkprodct.csh
+++ b/cspice/src/csupport/mkprodct.csh
@@ -150,7 +150,7 @@ else
 #
 #     -fPIC              position-independent code
 #
-    set TKCOMPILEOPTIONS = "-m64 -c -ansi -O2 -fPIC -DNON_UNIX_STDIO"
+    set TKCOMPILEOPTIONS = "-c -ansi -O2 -fPIC -DNON_UNIX_STDIO"
     echo " "
     echo "      Setting default compile options:"
     echo "      $TKCOMPILEOPTIONS"
@@ -161,7 +161,7 @@ if ( $?TKLINKOPTIONS ) then
     echo "      Using link options: "
     echo "      $TKLINKOPTIONS"
 else
-    set TKLINKOPTIONS = "-m64 -lm"
+    set TKLINKOPTIONS = "-lm"
     echo " "
     echo "      Setting default link options:"
     echo "      $TKLINKOPTIONS"
diff --git a/cspice/src/dskbrief_c/mkprodct.csh b/cspice/src/dskbrief_c/mkprodct.csh
index b98c1be..ce7aca9 100755
--- a/cspice/src/dskbrief_c/mkprodct.csh
+++ b/cspice/src/dskbrief_c/mkprodct.csh
@@ -145,7 +145,7 @@ else
 #                        implementation
 #
 #
-    set TKCOMPILEOPTIONS = "-m64 -c -ansi -O2 -DNON_UNIX_STDIO"
+    set TKCOMPILEOPTIONS = "-c -ansi -O2 -DNON_UNIX_STDIO"
     echo " "
     echo "      Setting default compile options:"
     echo "      $TKCOMPILEOPTIONS"
@@ -156,7 +156,7 @@ if ( $?TKLINKOPTIONS ) then
     echo "      Using link options: "
     echo "      $TKLINKOPTIONS"
 else
-    set TKLINKOPTIONS = "-m64 -lm"
+    set TKLINKOPTIONS = "-lm"
     echo " "
     echo "      Setting default link options:"
     echo "      $TKLINKOPTIONS"
diff --git a/cspice/src/dskexp_c/mkprodct.csh b/cspice/src/dskexp_c/mkprodct.csh
index b98c1be..ce7aca9 100755
--- a/cspice/src/dskexp_c/mkprodct.csh
+++ b/cspice/src/dskexp_c/mkprodct.csh
@@ -145,7 +145,7 @@ else
 #                        implementation
 #
 #
-    set TKCOMPILEOPTIONS = "-m64 -c -ansi -O2 -DNON_UNIX_STDIO"
+    set TKCOMPILEOPTIONS = "-c -ansi -O2 -DNON_UNIX_STDIO"
     echo " "
     echo "      Setting default compile options:"
     echo "      $TKCOMPILEOPTIONS"
@@ -156,7 +156,7 @@ if ( $?TKLINKOPTIONS ) then
     echo "      Using link options: "
     echo "      $TKLINKOPTIONS"
 else
-    set TKLINKOPTIONS = "-m64 -lm"
+    set TKLINKOPTIONS = "-lm"
     echo " "
     echo "      Setting default link options:"
     echo "      $TKLINKOPTIONS"
diff --git a/cspice/src/frmdif_c/mkprodct.csh b/cspice/src/frmdif_c/mkprodct.csh
index 3e49d98..88a43ac 100755
--- a/cspice/src/frmdif_c/mkprodct.csh
+++ b/cspice/src/frmdif_c/mkprodct.csh
@@ -145,7 +145,7 @@ else
 #                        implementation
 #
 #
-    set TKCOMPILEOPTIONS = "-m64 -c -ansi -O2 -DNON_UNIX_STDIO"
+    set TKCOMPILEOPTIONS = "-c -ansi -O2 -DNON_UNIX_STDIO"
     echo " "
     echo "      Setting default compile options:"
     echo "      $TKCOMPILEOPTIONS"
@@ -156,7 +156,7 @@ if ( $?TKLINKOPTIONS ) then
     echo "      Using link options: "
     echo "      $TKLINKOPTIONS"
 else
-    set TKLINKOPTIONS = "-m64 -lm"
+    set TKLINKOPTIONS = "-lm"
     echo " "
     echo "      Setting default link options:"
     echo "      $TKLINKOPTIONS"
diff --git a/cspice/src/inspkt_c/mkprodct.csh b/cspice/src/inspkt_c/mkprodct.csh
index 3e49d98..88a43ac 100755
--- a/cspice/src/inspkt_c/mkprodct.csh
+++ b/cspice/src/inspkt_c/mkprodct.csh
@@ -145,7 +145,7 @@ else
 #                        implementation
 #
 #
-    set TKCOMPILEOPTIONS = "-m64 -c -ansi -O2 -DNON_UNIX_STDIO"
+    set TKCOMPILEOPTIONS = "-c -ansi -O2 -DNON_UNIX_STDIO"
     echo " "
     echo "      Setting default compile options:"
     echo "      $TKCOMPILEOPTIONS"
@@ -156,7 +156,7 @@ if ( $?TKLINKOPTIONS ) then
     echo "      Using link options: "
     echo "      $TKLINKOPTIONS"
 else
-    set TKLINKOPTIONS = "-m64 -lm"
+    set TKLINKOPTIONS = "-lm"
     echo " "
     echo "      Setting default link options:"
     echo "      $TKLINKOPTIONS"
diff --git a/cspice/src/mkdsk_c/mkprodct.csh b/cspice/src/mkdsk_c/mkprodct.csh
index b98c1be..ce7aca9 100755
--- a/cspice/src/mkdsk_c/mkprodct.csh
+++ b/cspice/src/mkdsk_c/mkprodct.csh
@@ -145,7 +145,7 @@ else
 #                        implementation
 #
 #
-    set TKCOMPILEOPTIONS = "-m64 -c -ansi -O2 -DNON_UNIX_STDIO"
+    set TKCOMPILEOPTIONS = "-c -ansi -O2 -DNON_UNIX_STDIO"
     echo " "
     echo "      Setting default compile options:"
     echo "      $TKCOMPILEOPTIONS"
@@ -156,7 +156,7 @@ if ( $?TKLINKOPTIONS ) then
     echo "      Using link options: "
     echo "      $TKLINKOPTIONS"
 else
-    set TKLINKOPTIONS = "-m64 -lm"
+    set TKLINKOPTIONS = "-lm"
     echo " "
     echo "      Setting default link options:"
     echo "      $TKLINKOPTIONS"
diff --git a/cspice/src/mkspk_c/mkprodct.csh b/cspice/src/mkspk_c/mkprodct.csh
index 3e49d98..88a43ac 100755
--- a/cspice/src/mkspk_c/mkprodct.csh
+++ b/cspice/src/mkspk_c/mkprodct.csh
@@ -145,7 +145,7 @@ else
 #                        implementation
 #
 #
-    set TKCOMPILEOPTIONS = "-m64 -c -ansi -O2 -DNON_UNIX_STDIO"
+    set TKCOMPILEOPTIONS = "-c -ansi -O2 -DNON_UNIX_STDIO"
     echo " "
     echo "      Setting default compile options:"
     echo "      $TKCOMPILEOPTIONS"
@@ -156,7 +156,7 @@ if ( $?TKLINKOPTIONS ) then
     echo "      Using link options: "
     echo "      $TKLINKOPTIONS"
 else
-    set TKLINKOPTIONS = "-m64 -lm"
+    set TKLINKOPTIONS = "-lm"
     echo " "
     echo "      Setting default link options:"
     echo "      $TKLINKOPTIONS"
diff --git a/cspice/src/msopck_c/mkprodct.csh b/cspice/src/msopck_c/mkprodct.csh
index 3e49d98..88a43ac 100755
--- a/cspice/src/msopck_c/mkprodct.csh
+++ b/cspice/src/msopck_c/mkprodct.csh
@@ -145,7 +145,7 @@ else
 #                        implementation
 #
 #
-    set TKCOMPILEOPTIONS = "-m64 -c -ansi -O2 -DNON_UNIX_STDIO"
+    set TKCOMPILEOPTIONS = "-c -ansi -O2 -DNON_UNIX_STDIO"
     echo " "
     echo "      Setting default compile options:"
     echo "      $TKCOMPILEOPTIONS"
@@ -156,7 +156,7 @@ if ( $?TKLINKOPTIONS ) then
     echo "      Using link options: "
     echo "      $TKLINKOPTIONS"
 else
-    set TKLINKOPTIONS = "-m64 -lm"
+    set TKLINKOPTIONS = "-lm"
     echo " "
     echo "      Setting default link options:"
     echo "      $TKLINKOPTIONS"
diff --git a/cspice/src/spacit_c/mkprodct.csh b/cspice/src/spacit_c/mkprodct.csh
index 3e49d98..88a43ac 100755
--- a/cspice/src/spacit_c/mkprodct.csh
+++ b/cspice/src/spacit_c/mkprodct.csh
@@ -145,7 +145,7 @@ else
 #                        implementation
 #
 #
-    set TKCOMPILEOPTIONS = "-m64 -c -ansi -O2 -DNON_UNIX_STDIO"
+    set TKCOMPILEOPTIONS = "-c -ansi -O2 -DNON_UNIX_STDIO"
     echo " "
     echo "      Setting default compile options:"
     echo "      $TKCOMPILEOPTIONS"
@@ -156,7 +156,7 @@ if ( $?TKLINKOPTIONS ) then
     echo "      Using link options: "
     echo "      $TKLINKOPTIONS"
 else
-    set TKLINKOPTIONS = "-m64 -lm"
+    set TKLINKOPTIONS = "-lm"
     echo " "
     echo "      Setting default link options:"
     echo "      $TKLINKOPTIONS"
diff --git a/cspice/src/spkdif_c/mkprodct.csh b/cspice/src/spkdif_c/mkprodct.csh
index 3e49d98..88a43ac 100755
--- a/cspice/src/spkdif_c/mkprodct.csh
+++ b/cspice/src/spkdif_c/mkprodct.csh
@@ -145,7 +145,7 @@ else
 #                        implementation
 #
 #
-    set TKCOMPILEOPTIONS = "-m64 -c -ansi -O2 -DNON_UNIX_STDIO"
+    set TKCOMPILEOPTIONS = "-c -ansi -O2 -DNON_UNIX_STDIO"
     echo " "
     echo "      Setting default compile options:"
     echo "      $TKCOMPILEOPTIONS"
@@ -156,7 +156,7 @@ if ( $?TKLINKOPTIONS ) then
     echo "      Using link options: "
     echo "      $TKLINKOPTIONS"
 else
-    set TKLINKOPTIONS = "-m64 -lm"
+    set TKLINKOPTIONS = "-lm"
     echo " "
     echo "      Setting default link options:"
     echo "      $TKLINKOPTIONS"
diff --git a/cspice/src/spkmrg_c/mkprodct.csh b/cspice/src/spkmrg_c/mkprodct.csh
index 3e49d98..88a43ac 100755
--- a/cspice/src/spkmrg_c/mkprodct.csh
+++ b/cspice/src/spkmrg_c/mkprodct.csh
@@ -145,7 +145,7 @@ else
 #                        implementation
 #
 #
-    set TKCOMPILEOPTIONS = "-m64 -c -ansi -O2 -DNON_UNIX_STDIO"
+    set TKCOMPILEOPTIONS = "-c -ansi -O2 -DNON_UNIX_STDIO"
     echo " "
     echo "      Setting default compile options:"
     echo "      $TKCOMPILEOPTIONS"
@@ -156,7 +156,7 @@ if ( $?TKLINKOPTIONS ) then
     echo "      Using link options: "
     echo "      $TKLINKOPTIONS"
 else
-    set TKLINKOPTIONS = "-m64 -lm"
+    set TKLINKOPTIONS = "-lm"
     echo " "
     echo "      Setting default link options:"
     echo "      $TKLINKOPTIONS"
diff --git a/cspice/src/tobin_c/mkprodct.csh b/cspice/src/tobin_c/mkprodct.csh
index 3e49d98..88a43ac 100755
--- a/cspice/src/tobin_c/mkprodct.csh
+++ b/cspice/src/tobin_c/mkprodct.csh
@@ -145,7 +145,7 @@ else
 #                        implementation
 #
 #
-    set TKCOMPILEOPTIONS = "-m64 -c -ansi -O2 -DNON_UNIX_STDIO"
+    set TKCOMPILEOPTIONS = "-c -ansi -O2 -DNON_UNIX_STDIO"
     echo " "
     echo "      Setting default compile options:"
     echo "      $TKCOMPILEOPTIONS"
@@ -156,7 +156,7 @@ if ( $?TKLINKOPTIONS ) then
     echo "      Using link options: "
     echo "      $TKLINKOPTIONS"
 else
-    set TKLINKOPTIONS = "-m64 -lm"
+    set TKLINKOPTIONS = "-lm"
     echo " "
     echo "      Setting default link options:"
     echo "      $TKLINKOPTIONS"
diff --git a/cspice/src/toxfr_c/mkprodct.csh b/cspice/src/toxfr_c/mkprodct.csh
index 3e49d98..88a43ac 100755
--- a/cspice/src/toxfr_c/mkprodct.csh
+++ b/cspice/src/toxfr_c/mkprodct.csh
@@ -145,7 +145,7 @@ else
 #                        implementation
 #
 #
-    set TKCOMPILEOPTIONS = "-m64 -c -ansi -O2 -DNON_UNIX_STDIO"
+    set TKCOMPILEOPTIONS = "-c -ansi -O2 -DNON_UNIX_STDIO"
     echo " "
     echo "      Setting default compile options:"
     echo "      $TKCOMPILEOPTIONS"
@@ -156,7 +156,7 @@ if ( $?TKLINKOPTIONS ) then
     echo "      Using link options: "
     echo "      $TKLINKOPTIONS"
 else
-    set TKLINKOPTIONS = "-m64 -lm"
+    set TKLINKOPTIONS = "-lm"
     echo " "
     echo "      Setting default link options:"
     echo "      $TKLINKOPTIONS"
diff --git a/cspice/src/versn_c/mkprodct.csh b/cspice/src/versn_c/mkprodct.csh
index 3e49d98..88a43ac 100755
--- a/cspice/src/versn_c/mkprodct.csh
+++ b/cspice/src/versn_c/mkprodct.csh
@@ -145,7 +145,7 @@ else
 #                        implementation
 #
 #
-    set TKCOMPILEOPTIONS = "-m64 -c -ansi -O2 -DNON_UNIX_STDIO"
+    set TKCOMPILEOPTIONS = "-c -ansi -O2 -DNON_UNIX_STDIO"
     echo " "
     echo "      Setting default compile options:"
     echo "      $TKCOMPILEOPTIONS"
@@ -156,7 +156,7 @@ if ( $?TKLINKOPTIONS ) then
     echo "      Using link options: "
     echo "      $TKLINKOPTIONS"
 else
-    set TKLINKOPTIONS = "-m64 -lm"
+    set TKLINKOPTIONS = "-lm"
     echo " "
     echo "      Setting default link options:"
     echo "      $TKLINKOPTIONS"
